controllers/provider-gcp/pkg/controller/controlplane/valuesprovider.go

```diff
@@ -31,13 +31,17 @@ import (
 	"github.com/gardener/gardener/pkg/operation/common"
 	"github.com/gardener/gardener/pkg/utils/chart"
 	"github.com/gardener/gardener/pkg/utils/secrets"
+
```

Comment:
Are we following a convention for ordering imports? I see you group into multiple groups, while I tend to use only 3 groups (standard library, project, other), while other people do something different. Perhaps we should standardize?
Ordering in 3 (or less) groups has the advantage it can be automated and enforced by `goimports`. I don't think it's possible with more than 3 groups.

Reply:
We have not agreed on any of such convention - I'm just personally following it :-) If the `goimports` can automate this then we probably should follow its convention. I'll keep it in mind for the future. 👍